src/test/scala/com/webtrends/harness/component/akkahttp/RouteGeneratorTest.scala

```diff
@@ -226,7 +227,16 @@ class RouteGeneratorTest extends WordSpec with ScalatestRouteTest with Predefine
 
   "Cors settings" should {
     val whiteListOrigin = HttpOrigin("http://example.com")
-    val corsSettings = CorsSettings.defaultSettings
+    val corsSettings = CorsSettings(ConfigFactory.parseString(
```

Comment:
There is no need to use json or typesafe config here.

You can just use the CorsSettings constructor or helper "with" methods to construct it.

Reply:
Updated with empty list